jigglypuffRL/deeprl/dqn/dqn.py

```diff
@@ -171,7 +180,34 @@ def create_model(self, network_type):
                         setattr(self, key, item)
                 print("Loaded pretrained model")
 
-            self.target_model = deepcopy(self.model)
+        elif self.network_type == "cnn":
+            if self.history_length is None:
+                self.history_length = 4
+
+            if self.transform is None:
+                self.transform = transforms.Compose([
+                    transforms.ToPILImage(),
+                    transforms.Grayscale(),
+                    transforms.Resize((110, 84)),
+                    transforms.CenterCrop(84),
```

Comment:
Fixing the image size to (110, 84) might not work well for environments with larger image sizes? (non-Atari)? We need to figure out a cleaner way to make it env agnostic. Any opinions @Sharad24 @Het-Shah ?

Reply:
Lets just keep transform as a user passable arg for now?

Reply:
Okay, let's do that

Reply:
Yeah, I just used it to test the working. The transform can be user modified.